### DISCUSSION TITLE
Is there any way to get List of Opened Streaming connections List

### DISCUSSION THREAD

**Discussion by yashnb-serverpod** [serverpod/serverpod#1175] [Q&A] [answered]
How can i create serverside local list variable in which i can store 

**List<Map<String,StreamingSession>> openedConnection;**

where value will be **[{userId_timestamp : StreamingSession instance}]**

Basically I want to get sessions which are opened and i can **sendStreamMessage** on the session fetched by userId.

@vlidholt

**Comment by vlidholt** [comment, marked as answer]
The better way to do this is to start listening to a channel in the messaging central when a user connects. Have a look at https://pixorama.live for a full example.

**Reply by yashnb-serverpod** [reply]
yes but how we can fetch opened session, is there is serverpod default method or need to manage list of opened sessions manually?

**Reply by vlidholt** [reply]
You would need to manually keep such a list, but for almost all use cases it's not needed.
